result.go

```diff
@@ -356,10 +357,11 @@ func (q Query) ToMap(result interface{}) {
 		})
 }
 
-// ToMapBy iterates over a collection and populates result map with elements.
+// ToMapBy iterates over a collection and populates the result map with elements.
 // Functions keySelector and valueSelector are executed for each element of the collection
 // to generate key and value for the map. Generated key and value types must be assignable
 // to the map's key and value types.
+// ToMapBy doesn't empty the result map before populating it.
```

Comment:
if you want to make these new paragraphs, you need to have an empty comment line before the text.